source/adapters/hip/program.cpp

```diff
@@ -259,6 +412,16 @@ UR_APIEXPORT ur_result_t UR_APICALL urProgramCreateWithBinary(
 
   // TODO: Set metadata here and use reqd_work_group_size information.
   // See urProgramCreateWithBinary in CUDA adapter.
+  if (pProperties) {
```

Comment:
@jchlanda I was just wondering whether this affects your work on `reqd_work_group_size` metadata?

Reply:
Yeap, I've added essentially the same snippet to set the metadata.
I guess we should be OK though, whichever patch gets in first will cause a merge conflict for the other, but that should be easy to resolve.

Reply:
Agreed, git will likely resolve it for us. Not to worry.